### ISSUE TITLE
Can you please ignore `20j` ?

### ISSUE THREAD

**Issue by gukz** [takac/vim-hardtime#63]
I want I can use duplicated `20j` commands. What do you think?

**Comment by fakeharxy** [comment]
yeah just to second this old request, it would be great if all `j`s and `k`s with counts are ignored? It's annoying to go `18j` and then have to wait for a second because you missed where you were aiming by one!

**Comment by matjansen** [comment]
Works for me with [hardtime_motion_with_count_resets](https://github.com/takac/vim-hardtime#reset-counter-if-motion-is-used-with-a-count).
